### ISSUE TITLE
Aparrent conflict with BetterRoleColor

### ISSUE THREAD

**Issue by nicholasalexanderholmgren** [estroBiologist/pluralchum#36]
Enabling both [BetterRoleColors ](https://github.com/rauenzi/BetterDiscordAddons)by Zerebos and this plugin results proxy'd text appearing as white, rather than the color of the proxy's set user color. Only this plugin, [ZeresPluginLibrary](https://github.com/rauenzi/BDPluginLibrary), and BetterRoleColors are enabled. Including configs for all three plugins for hopeful ease of replication.

[Pluralchum.config.json](https://github.com/estroBiologist/pluralchum/files/12884522/Pluralchum.config.json)
[ZeresPluginLibrary.config.json](https://github.com/estroBiologist/pluralchum/files/12884523/ZeresPluginLibrary.config.json)
[BetterRoleColors.config.json](https://github.com/estroBiologist/pluralchum/files/12884524/BetterRoleColors.config.json)

**Comment by SnowboundCabin** [comment]
this can be fixed by adding `+ " !important"` to the color property in the function `setMessageTextColour`. BRC will attempt to override but unless also set to use !important, pluralchum's will be treated as important and used.

**Comment by ariagivens** [comment]
I couldn't replicate the issue with the latest versions of Pluralchum and BetterRoleColors. Can you confirm if it's still an issue?

**Comment by SnowboundCabin** [comment]
Yes, on Pluralchum 2.2.0 and BRC 0.10.2 (both latest as of writing) there is still an issue, although it's that with PC enabled, BRC will not recolor non-PK message tect.
